go/cmd/dolt/commands/login.go

```diff
@@ -243,7 +243,7 @@ func loginWithCreds(ctx context.Context, dEnv *env.DoltEnv, dc creds.DoltCreds,
 
 func openBrowserForCredsAdd(dc creds.DoltCreds, loginUrl string) {
 	url := fmt.Sprintf("%s#%s", loginUrl, dc.PubKeyBase32Str())
-	cli.Printf("Opening a browser to:\n\t%s\nPlease associate your key with your account.\n", url)
+	cli.Printf("Attempting to automatically open the credentials page in your default browser.\nIf the browser does not open or you wish to use a different device to authorize this request, open the following URL:\n\t%s\nPlease associate your key with your account.\n", url)
```

Comment:
Can you break into multiple PrintFs so we don't need to search for `\n` in the text.